### PR TITLE
Added quarkus-jms-spi to quarkus bom

### PR DESCRIPTION
**Pull request by turing85**

### CODE REVIEW

**Comment by turing85** [comment]
Companion PR for `2.16`: https://github.com/quarkusio/quarkus/pull/31075

File: bom/application/pom.xml [hunk 1]
```diff
@@ -1292,6 +1292,19 @@
                 <artifactId>quarkus-jdbc-derby-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jms-spi</artifactId>
+                <version>${project.version}</version>
```
Comment (zhfeng) [inline bom/application/pom.xml:1298]: @turing85 I think we don't have `io.quarkus:quarkus-jms-spi`.

**Review by zhfeng — CHANGES_REQUESTED** [review]

File: bom/application/pom.xml [hunk 2]
```diff
@@ -1292,6 +1292,19 @@
...
+            </dependency><dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jms-spi-deployment</artifactId>
+                <version>${project.version}</version>
+            </dependency><dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jms-spi-parent</artifactId>
```
Comment (zhfeng) [inline bom/application/pom.xml:1305]: And no need to include `io.quarkus:quarkus-jms-spi-parent`

**Review by zhfeng — CHANGES_REQUESTED** [review]

File: bom/application/pom.xml (hunk 1 above)
Comment (turing85) [reply bom/application/pom.xml]: true. Removed (also for the companion PR).

**Review by zhfeng — APPROVED** [review]

**Review by gsmet — APPROVED** [review]
Ah yes, that was an oversight.

**Comment by turing85** [comment]
@gsmet a timely backport would be much appreciated since we have an open PR [here](https://github.com/quarkiverse/quarkus-artemis/issues/148) that - for now - defines the version explicitly. We would like to resolve this in a timely manner so we do not forget it.

**Pull request merged**